### ISSUE TITLE
Quality adjusted urban land prices time series

### ISSUE THREAD

**Issue by rgrp** [datasets/registry#78]
Urban land prices (price per sq m or per sq ft) for e.g. major cities over a reasonable span of time. Would like to quality adjust if possible (just for internal quality - not improvements in transportation or similar).

**Comment by rgrp** [comment]
@Yannael do you want to see if we can rustle up someone to take this on?

**Comment by Yannael** [comment]
@rgrp Yes, any suggestion where the data can be collected from?

**Comment by rgrp** [comment]
@Yannael no, so first step would be research (aka googling) - no data wrangling needed!
